src/libs/ReportUtils.ts

```diff
@@ -4138,12 +4138,13 @@ function getOriginalReportID(reportID: string, reportAction: OnyxEntry<ReportAct
 }
 
 /**
- * Return the pendingAction and the errors we have when creating a chat or a workspace room offline
+ * Return the pendingAction and the errors we have when creating a chat or a workspace room offline or pay the request
  */
 function getReportOfflinePendingActionAndErrors(report: OnyxEntry<Report>): ReportOfflinePendingActionAndErrors {
-    // We are either adding a workspace room, or we're creating a chat, it isn't possible for both of these to be pending, or to have errors for the same report at the same time, so
+    // We are either adding a workspace room, or we're creating a chat, or we're paying request money, it isn't possible for both of these to be pending, or to have errors for the same report at the same time, so
```

Comment:
```
// It shouldn't be possible for all of these actions to be pending (or to have errors) for the same report at the same time, so just take the first that exists
```

Reply:
I just suggest to improve the language here. An important part is that you kept the word "both", while we add a _third_ case. I suggest to drop the "both" and shorten the comment, making it more generic.

Another thing is whether this statement is actually true (especially, after our changes). To be honest, it isn't completely clear to me why these actions cannot be pending at the same time.